docs/dapp/sapphire/README.mdx

```diff
@@ -13,91 +14,89 @@ Sapphire allows for:
-          blockExplorerUrls: ['https://explorer.sapphire.oasis.io'],
-        },
-      ],
-    })
-  }}
->
-  Click here to register Sapphire Mainnet to your MetaMask or Brave Wallet
-</button>
+- Network name: `sapphire`
+- Long network name: `Oasis Sapphire`
+- Chain ID:
+  - Hex: `0x5afe`
+  - Decimal: `23294`
 
 ### Testnet
 
-* RPC HTTP endpoint: `https://testnet.sapphire.oasis.dev`
-* RPC WebSockets endpoint: `wss://testnet.sapphire.oasis.dev/ws`
-* Chain ID:
-  * Hex: 0x5aff
-  * Decimal: 23295
-* Block explorer: [https://testnet.explorer.sapphire.oasis.dev](https://testnet.explorer.sapphire.oasis.dev)
-
-<button
-  class="button button--primary margin-bottom--md"
-  onClick={() => {
-    if (!window.ethereum?.request) {
-      return alert('Have you installed MetaMask yet? If not, please do so.\n\nComputer: Once it is installed, you will be able to add the ParaTime to your MetaMask.\n\nPhone: Open the website through your MetaMask Browser to add the ParaTime.')
-    }
-    window.ethereum.request({
-      method: 'wallet_addEthereumChain',
-      params: [
-        {
-          chainId: '0x5aff',
-          chainName: 'Oasis Sapphire Testnet',
-          nativeCurrency: {
-            name: 'TEST',
-            symbol: 'TEST',
-            decimals: 18,
-          },
-          rpcUrls: ['https://testnet.sapphire.oasis.dev/', 'wss://testnet.sapphire.oasis.dev/ws'],
-          blockExplorerUrls: ['https://testnet.explorer.sapphire.oasis.dev'],
-        },
-      ],
-    })
-  }}
->
-  Click here to register Sapphire Testnet to your MetaMask or Brave Wallet
-</button>
+- Network name: `sapphire-testnet`
+- Long network name: `Oasis Sapphire Testnet`
+- Chain ID:
+  - Hex: `0x5aff`
+  - Decimal: `23295`
+
+## RPC Endpoints
+
+:::danger
+
+The RPC endpoint is a *point of trust*. Beside traffic rate limiting, it can
+also perform censorship or even a man-in-the-middle attack. If you have security
+considerations, we strongly recommend that you set up your own [ParaTime client
+node][paratime-client-node] and the [Web3-compatible gateway].
+
+:::
+
+[Web3-compatible gateway]: ../../node/web3.md
+[paratime-client-node]: ../../node/run-your-node/paratime-client-node.mdx
+
+You can connect to one of the public Web3 gateways below (in alphabetic order):
+
+| Provider                    | Mainnet RPC URLs                                                        | Testnet RPC URLs                                                                           | Supports Confidential Queries |
+|-----------------------------|-------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|-------------------------------|
+| [1RPC]                      | <S rpcs={['https://1rpc.io/oasis/sapphire']} />                         | N/A                                                                                        | Yes                           |
```

Comment:
This doesn't seem to be working, requests in background page in chrome://extensions/?id=nkbihfbeogaeaoehlefnkodbefgpgknn are failing with `"Failed to validate quota usage"`

Reply:
notes from trying to use a free account:
- their dashboard is very slow
- using my newly generated `1rpc.io/4n**************/oasis/sapphire` kept failing for a few minutes
- I clicked rotate key to generate new, refreshed very slowly, re-added it to MetaMask
- it still failed for a few minutes with `1RPC+ api_key is invalid. Re-add the network if you have rotated your api_key`
- successfully loaded a few times
- failing `1RPC+ api_key is invalid. Re-add the network if you have rotated your api_key` again
- used 20 / 20000 of daily free relays
- reloaded the dashboard, it says used 0 / 20000 again :shrug: